docs/data-structures/inordertraversal.mdx

```diff
@@ -0,0 +1,206 @@
+---
+title: Inorder traversal of Binary Tree
+slug: /data-structures/inordertraversal
+---
+
+## Introduction
+
+Binary Tree is a type of tree which have atmost two children or we can say atmost degree 2. So to traverse the binary tree according to Depth first search. We have a traversal algorithm called Inorder traversal.
+
+## Explanation 
+In this type of traversal(Inorder traversal). First we have to traverse the left part then its root then right part respectively. In left part we have to traverse whole left subtree after that root of thet subtree and then whole right subtree. Inorder: (left->root->right). 
```

Comment:
Make it:

In this type of traversal(Inorder traversal), first, we have to traverse the left part then its root then the right part respectively. In the left part, we have to traverse the whole left subtree after that root of that subtree and then the whole right subtree. Inorder: (left->root->right).